nifi-nar-bundles/nifi-graph-bundle/nifi-graph-processors/pom.xml

```diff
@@ -84,11 +84,6 @@
             <version>1.14.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
-        <dependency>
```

Comment:
[WARNING] Some problems were encountered while building the effective model for org.apache.nifi:nifi-graph-processors:jar:1.14.0-SNAPSHOT
[WARNING] 'dependencies.dependency.(groupId:artifactId:type:classifier)' must be unique: org.apache.nifi:nifi-record-serialization-service-api:jar -> version (?) vs 1.14.0-SNAPSHOT @ line 98, column 21